custom_components/dual_smart_thermostat/const.py

```diff
@@ -79,6 +79,7 @@
 ATTR_PREV_HUMIDITY = "prev_humidity"
 ATTR_HVAC_ACTION_REASON = "hvac_action_reason"
 ATTR_TIMEOUT = "timeout"
```

Comment:
Could you rename the constant to ATTR_OPENING_TIMEOUT?

Reply:
Yes, of course. Only the alias or also the real name "timeout" (but in this case would be a breaking change)?